### PR TITLE
Fix releasing script;

### PR DESCRIPTION
**Pull request by ShengHuaWu**
### Description
The `bundle exec fastlane ios release` is going to do the followings in order
- Ensure there is no uncommitted changes.
- Fetch the latest release version from private SDK.
- Download the `.framework` files from the release of private SDK.
- Update the changelog according to the changelog of private SDK.
- Bump the `podspec` version number.
- Create a new tag and push it to remote.
- Push the master branch to remote.
- Create the sample project.
- Create a new release on GitHub.
- Push `podspec` files to remote.

Please help me to verify the releasing document. 😄 
https://github.com/conichiGMBH/conichi-ios-sdk/blob/CNI-9324/RELEASING.md

### CODE REVIEW

**Comment by conichi-ci-public** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">fastlane/Fastfile modified</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by ShengHuaWu** [comment]
I mark this PR as [WIP], because I haven't tested yet.
I am going to test it after there is a new version of private SDK.

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+### Prepare
```
Comment (jotshin) [inline RELEASING.md]: Thanks for the adding of these. 😄

**Review by jotshin — APPROVED** [review]
Thanks for the great job. 👍

**Comment by ShengHuaWu** [comment]
@David-Henner @vincentjacquesson 

I'm going to merge this, in order to let @jotshin test the releasing flow.
If there is anything needs changing, please inform me and I will create another ticket. 😄

**Pull request merged**